src/main/java/uk/ac/ebi/intact/app/internal/model/core/network/Network.java

```diff
@@ -45,17 +60,21 @@ public class Network implements AddedEdgesListener, AboutToRemoveEdgesListener,
     private final Map<CyEdge, EvidenceEdge> evidenceEdges = new HashMap<>();
 
     private final Set<String> taxIds = new HashSet<>();
+    @Getter
     private final Set<String> interactorTypes = new HashSet<>();
     private final Map<String, String> speciesNameToId = new HashMap<>();
     private final Map<String, String> speciesIdToName = new HashMap<>();
 
+    private final CyGroupFactory groupFactory;
+    private CyGroupManager groupManager;
+
+    @Getter
+    boolean areGroupCollapsed = false;
```

Comment:
How is the Lombok getter named for this variable? Is it `isAreGroupCollapsed`? If so, let's rename the variable to `groupsCollapsed`. Do we need the getter anyway?